features/projects/src/test/kotlin/com/agilefreaks/freaks_catalog/features/projects/CoroutineRule.kt

```diff
@@ -0,0 +1,25 @@
+package com.agilefreaks.freaks_catalog.features.projects
```

Comment:
This seems to be copy/paste code? I have a deja vu I've seen this code already in our app